packages/content-management/content_editor/src/components/inspector_flyout_content.test.tsx

```diff
@@ -104,32 +104,37 @@ describe('<ContentEditorFlyoutContent />', () => {
       expect(find('saveButton').text()).toBe('Update foo');
     });
 
-    test('should send back the updated item to the onSave() handler', async () => {
+    test('should save form only if something changes', async () => {
       const onSave = jest.fn();
 
       await act(async () => {
         testBed = await setup({ onSave, isReadonly: false });
       });
 
-      const {
-        find,
-        component,
-        form: { setInputValue },
-      } = testBed!;
-
-      await waitForValidationResults();
+      const { find, component } = testBed!;
 
       await act(async () => {
         find('saveButton').simulate('click');
       });
 
-      expect(onSave).toHaveBeenCalledWith({
```

Comment:
Removed this assertion, because now due to a disabled state for the button, if there were not any changes made, it is impossible to click on it.
So now firstly, the form values get updated (line 139/140) and then clicking to test the onSave handler.